smartsim/database/orchestrator.py

```diff
@@ -590,10 +605,14 @@ def _build_batch_settings(
         batch: bool,
         account: str,
         time: str,
+        *,
```

Comment:
Discussed offline and decided to keep the `*` to prevent `launcher` from potentially being treated as a positional arg, thus preserving the existing API